docusaurus/docusaurus.config.js

```diff
@@ -186,7 +186,9 @@ const config = {
   ],
   customFields: {
     requestErdApiUrl: process.env.REQUEST_ERD_API_URL,
-    markpromptProjectKey: process.env.MARKPROMPT_PROJECT_KEY,
+    markpromptProjectKey:
+      process.env.MARKPROMPT_PROJECT_KEY ||
+      "sk_test_cbPFAzAxUvafRj6l1yjzrESu0bRpzQGK",
```

Comment:
@ian-at-airbyte I opted for setting a default instead of having our public prod key directly here. this way local development should work and we can still use/test this feature in development (which wouldn't work if I'd set directly the prod key here)